### ISSUE TITLE
Regarding training on CelebA-HQ-256 dataset

### ISSUE THREAD

**Issue by parthpatel002** [rosinality/style-based-gan-pytorch#93]
I am trying to train StyleGAN on CelebA-HQ-256 dataset. The PyTorch FID decreases continuously as progressive growing reaches 128 resolution (it achieves around FID 19 for 128 resolution). But when it progresses to 256 resolution, FID increases to around 35 and oscillates between 35 and 25 with further training on 256 resolution. I have trained the model till around 300,000 iterations.

Can you please specify the configuration (i.e. hyperparameter choices) to be used for CelebA-HQ-256 dataset? Also, can you please suggest the approximate no. of iterations the model should be trained on?

For reference, I am using PyTorch FID calculation from [this repo](https://github.com/ajbrock/BigGAN-PyTorch).

**Comment by rosinality** [comment]
Sorry, but I haven't tried to train this model on celebA-HQ. But maybe you need to adjust learning rate schedules for image resolutions: https://github.com/rosinality/style-based-gan-pytorch/blob/master/train.py#L333
